src/main/java/ldbc/snb/datagen/util/LdbcConfiguration.java

```diff
@@ -56,7 +56,7 @@ public float getFloat(String name, float defaultValue) {
     private String getHexDigits(String value) {
         boolean negative = false;
         String str = value;
-        String hexString = null;
+        String hexString;
```

Comment:
my IDE reports several code quality issues in these functions, but I was reluctant to change anything because i copy pasted from hadoop and i think it's easier if we don't change anything